src/agent/inspectordebugapi.ts

```diff
@@ -394,11 +439,13 @@ export class InspectorDebugApi implements debugapi.DebugApi {
       } else {
         const frame = callFrames[0];
         const evaluatedExpressions = breakpoint.expressions.map(function(exp) {
-          const result = state.evaluate(exp, frame, that.v8Inspector);
+          const result = state.evaluate(exp, frame, that.v8Inspector, true);
           // TODO: Address the case where `result.mirror` is `undefined`.
-          return result.error ?
-              result.error :
-              (result.object as inspector.Runtime.RemoteObject).value();
+          if (result.error)
```

Comment:
To reduce the chances of introducing subtle bugs, `if` statements should use braces, even if the body contains a single statement.

Reply:
Done.